Mathlib/CategoryTheory/Monoidal/Cartesian/Grp_.lean

```diff
@@ -182,14 +183,61 @@ lemma GrpObj.one_inv : η[G] ≫ ι = η := by simp [GrpObj.inv_eq_inv, GrpObj.c
 
 @[deprecated (since := "2025-09-13")] alias Grp_Class.inv_eq_inv := GrpObj.inv_eq_inv
 
-instance [BraidedCategory C] [IsCommMonObj G] : IsMonHom ι[G] where
+variable [BraidedCategory C]
+
+instance [IsCommMonObj G] : IsMonHom ι[G] where
   one_hom := by simp [one_eq_one, ← Hom.inv_def]
   mul_hom := by simp [GrpObj.mul_inv_rev]
 
 attribute [local simp] Hom.inv_def in
-instance [BraidedCategory C] [IsCommMonObj G] {f : M ⟶ G} [IsMonHom f] : IsMonHom f⁻¹ where
+instance [IsCommMonObj G] {f : M ⟶ G} [IsMonHom f] : IsMonHom f⁻¹ where
+
+namespace Grp_
+variable {G H : Grp_ C} [IsCommMonObj H.X]
+
+-- TODO: Make `Grp_.toMon` an abbrev in mathlib.
+set_option allowUnsafeReducibility true in
+attribute [reducible] Grp_.toMon
```

Comment:
Can you use `unseal ... in` instead?

Reply:
Nope, that doesn't work:
```
unseal Grp_.toMon in
```
throws
```
failed to set `[local semireducible]`, `toMon` is currently `[semireducible]`, `[irreducible]` expected

Note: Use `set_option allowUnsafeReducibility true` to override reducibility status validation
```
and doesn't even fix the error below (which is that `η[H.toMon]` throws `failed to synthesize MonObj H.toMon`).